_locales/_languages.json

```diff
@@ -32,4 +37,4 @@
   "uk": "Українська",
   "zh_CN": "简体中文",
   "zh_TW": "繁體中文"
-}
+}
```

Comment:
is that the order how it appears in the ui? if so, i would change to to alphabetic order inside the latin alphabet, followed by the other alphabets.

that is probably also not perfect, however, a bit less random (eg. in 1.15 the two cyrillic languages appear at quite different positions, without an obvious rule).

maybe just use the same order as on android.

Reply:
no, the sorting is done at runtime in https://github.com/deltachat/deltachat-desktop/blob/master/src/main/menu.ts#L32